docs/guides/cloud/github-integration.mdx

```diff
@@ -150,8 +145,23 @@ installation if you choose <strong>All repositories</strong>.
 
 :::
 
+{' '}
```

Comment:
Sometimes prettier tries to create empty white space in jsx ( {' '} ) if it interprets whitespace in the area in question. The problem is if its superfluous it can end up rendering that code in the browser. You can eliminate this by just making sure there aren't extra lines here and re-lint and this should go away.